ex4/tema4.js

```diff
@@ -0,0 +1,14 @@
+
+
+
+  function addNumber(...theArgs)
```

Comment:
```suggestion
  function getSum(...theArgs)
```
un nume mai sugestiv ar fi "getSum"